### PR TITLE
Add content for first draft

### PR DESCRIPTION
**Pull request by thomastuts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,125 @@
...
+Let's quickly talk about what _state_ is in React. State is basically data that is mutated in your component. Like with any
+state, this state can also **change**. That's an important part: while a component can't change its own props, it _can_
+change its state.
+
+State is used to handle several things in your component:
+
+- Interactivity (e.g. changing data when a user clicks a button)
+- Fetching remote data (remote data is, by definition, not available right away when the component is mounted — state
+gives us a way of updating the component once that data arrives)
+- Reacting to the passing of time (i.e. setting an interval or timeout)
```
Comment (ezmiller) [inline README.md:18]: Nice succinct summary.

File: README.md [hunk 2]
```diff
@@ -1,68 +1,125 @@
...
-       {React.cloneElement(
-         this.props.children,
-         { ...c.props, on: this.state.on })}
-      </div>
-    )
+      <div className="toggle-button">I am toggled {this.state.isEnabled ? 'on' : 'off'}.</div>
+    );
   }
-})
+});
```
Comment (ezmiller) [inline README.md:73]: Good idea to show them the old method as well, as there's a good chance they'll run into that.

**Comment by ezmiller** [comment]
@thomastuts this looks good. Two main comments related to the minimal representation of state section and theme:
- When I read through that section, I saw the example of computed state, and thought precisely what you later explained: that this was an unnecessary usage. But the question occurred to me: why provide a bad example? I think I see what you were doing: undoing your own example as a way to teach two points, which is good because it piques the reader's attention by confounding expectations. Maybe, you could soften the the transition by saying something like: "While this is a working example of computed state, it's not a particularly good idea. Do you see why?" And then explain.
- A slightly broader point that I make very tentatively since you may well have dealt with it elsewhere has to do with explaining why state is to be avoided. For example, in the minimal state section you say, "It's important to try and keep your state as small as possible." Might it make sense to explain why --very briefly -- as a way to introduce the idea that state is hard to manage and that React is ideal for addressing that problem?

**Comment by thomastuts** [comment]
@ezmiller Great feedback, thanks! I've updated the content — let me know if you have any other thoughts!

**Pull request merged**